src/components/Software/index.tsx

```diff
@@ -60,9 +68,13 @@ export const SoftwareComponent = () => {
         subtitle="Versenden Sie Ihre Rechnungen direkt aus lexoffice per Brief an Ihre Kunden."
       />
       <Container>
-        <Row className="mt-5 mb-3 ">
-          <Col className="pb-4 justify-content-center">
-            <TitleImage className="img-fluid" />
+        <Row className="my-5">
+          <Col>
+            <Img
+              alt="Briefversand für lexoffice"
+              fluid={titleImage.childImageSharp.fluid}
+              fadeIn={false}
```

Comment:
Without this an unsharp version of the image fades in across 500ms which looks terrible for our title image 

Perhaps instead of fadeIn=false I should not be using gatsby-transformer-sharp in the first place